src/images/icons/Vector Globe Icon.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="53" height="52" viewBox="0 0 53 52" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M28.3102 0.0650931C28.0518 -0.00243305 27.7803 -0.0184745 27.5125 0.0208523C27.166 0.00697352 26.8187 -5.62993e-06 26.4707 -5.62993e-06C26.1228 -5.62993e-06 25.7756 0.00696923 25.4293 0.0208391C25.1615 -0.0184629 24.8901 -0.00243193 24.6318 0.0650521C18.4082 0.506082 12.5266 3.17454 8.08593 7.61522C3.20998 12.4912 0.470703 19.1044 0.470703 26C0.470703 29.4144 1.14321 32.7953 2.44984 35.9498C3.75646 39.1042 5.6716 41.9704 8.08593 44.3848C10.5002 46.7991 13.3665 48.7142 16.5209 50.0209C19.1077 51.0923 21.8467 51.7374 24.6315 51.9349C24.8899 52.0024 25.1614 52.0185 25.4294 51.9791C25.776 51.993 26.1232 52 26.4707 52C26.8183 52 27.1656 51.993 27.5124 51.9791C27.7804 52.0185 28.052 52.0024 28.3104 51.9348C31.0951 51.7373 33.8339 51.0922 36.4205 50.0209C39.5749 48.7142 42.4412 46.7991 44.8555 44.3848C47.2698 41.9704 49.185 39.1042 50.4916 35.9498C51.7982 32.7953 52.4707 29.4144 52.4707 26C52.4707 19.1044 49.7314 12.4912 44.8555 7.61522C40.4149 3.17468 34.5336 0.506247 28.3102 0.0650931ZM26.2631 4.00097C26.3323 4.00032 26.4015 3.99999 26.4707 3.99999C26.5401 3.99999 26.6094 4.00032 26.6787 4.00098C28.8885 7.75097 30.5118 11.7975 31.5085 16H21.4333C22.43 11.7975 24.0534 7.75096 26.2631 4.00097ZM21.3789 4.59727C17.4488 5.53237 13.8176 7.54039 10.9144 10.4436C9.26981 12.0882 7.91251 13.9663 6.87476 16H17.3321C18.186 12.0494 19.5441 8.21641 21.3789 4.59727ZM16.6455 20H5.30464C4.75602 21.9351 4.4707 23.9529 4.4707 26C4.4707 28.033 4.75247 30.0519 5.30469 32H16.6455C16.392 30.0169 16.2637 28.013 16.2637 26C16.2637 23.9869 16.392 21.9831 16.6455 20ZM17.3321 36H6.87478C7.92042 38.049 9.28265 39.9246 10.9144 41.5563C12.9572 43.5992 15.3825 45.2197 18.0517 46.3253C19.1329 46.7732 20.2456 47.133 21.3788 47.4026C19.5441 43.7835 18.186 39.9505 17.3321 36ZM26.2631 47.999C24.0534 44.249 22.43 40.2025 21.4333 36H31.5085C30.5118 40.2025 28.8885 44.249 26.6787 47.999C26.6094 47.9997 26.5401 48 26.4707 48C26.4015 48 26.3323 47.9997 26.2631 47.999ZM31.5631 47.4025C32.6962 47.1329 33.8087 46.7731 34.8897 46.3253C37.5589 45.2197 39.9842 43.5992 42.0271 41.5563C43.6588 39.9246 45.021 38.049 46.0666 36H35.6098C34.7559 39.9505 33.3978 43.7834 31.5631 47.4025ZM36.2964 32H47.6367C48.1889 30.0519 48.4707 28.033 48.4707 26C48.4707 23.9529 48.1854 21.9351 47.6368 20H36.2964C36.5498 21.9831 36.6782 23.9869 36.6782 26C36.6782 28.013 36.5498 30.0169 36.2964 32ZM32.2608 20H20.681C20.4041 21.9807 20.2637 23.9853 20.2637 26C20.2637 28.0147 20.4041 30.0193 20.681 32H32.2608C32.5377 30.0193 32.6782 28.0147 32.6782 26C32.6782 23.9853 32.5377 21.9807 32.2608 20ZM35.6098 16C34.7559 12.0495 33.3977 8.21649 31.563 4.59738C35.4929 5.53253 39.1239 7.5405 42.0271 10.4436C43.6716 12.0882 45.0289 13.9663 46.0666 16H35.6098Z" fill="#E9F0FB"/>
```

Comment:
It's better avoid using hardcoded colors for svg images. But let it be.

Reply:
fixed. made svg as react component and passed styles with colours using styled-component.